pallets/investments/src/lib.rs

```diff
@@ -975,9 +997,9 @@ where
 			.checked_add(
 				&fulfillment
 					.price
-					.reciprocal()
+					.reciprocal_floor()
```

Comment:
This is great! Maybe you can add a comment though, explaining why you are rounding down here?

Reply:
Yes, thats definetly needed. Will add that.